espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/drivers/ComposeDriver.kt

```diff
@@ -49,7 +49,10 @@ class ComposeDriver : AppDriver {
     private fun toNodeInteractionsCollection(params: Locator): SemanticsNodeInteractionCollection {
         val parentNodeInteraction = params.elementId?.let { getNodeInteractionById(it) }
         return parentNodeInteraction?.findDescendantNodeInteractions(params)
-            ?: EspressoServerRunnerTest.composeTestRule.onAllNodes(semanticsMatcherForLocator(params))
+            ?: EspressoServerRunnerTest.composeTestRule.onAllNodes(
+                useUnmergedTree = true,
```

Comment:
would it make to make this option configurable via settings?

Reply:
I thought about it, but it doesn't make sense to keep the merged tree, because, a merged node can have many texts (from children) and there is no good way to provide all of them to client

Reply:
ok, lets keep it as is for now. We can always add it if requested

Reply:
Yes, It's still very new and we are discovering many use cases during automation. If someone has a use case for merged tree, I will be happy to add it in settings api